src/Symfony/Component/Translation/Tests/TranslatorTest.php

```diff
@@ -449,6 +449,7 @@ public function getTransFileTests()
     public function getTransTests()
     {
         return [
+            ['', 'Empty string', '', [], 'fr', ''],
```

Comment:
isn't this testing it for the non-intl version ? Other messages are for the legacy format here.

Reply:
@stof this tests that there no exception is thrown with an empty message when you don't use `+intl-icu`. It was already working fine, I've just added a test for this corner case.

Reply:
According to @yceruto comment, I'll update the PR so this comment is no longer relevant